scripts/run_e2e_tests.py

```diff
@@ -286,7 +286,7 @@ def run_tests(args):
                     # Although our unit tests always provide unicode strings,
                     # the actual server needs this failsafe since it can output
                     # non-unicode strings.
-                    line = line.encode('utf-8')  # pragma: nocover
+                    line = line.encode('utf-8')  # pragma: no cover
```

Comment:
Why is this change needed?

Reply:
in entire code base `no cover` is used, but in 2 places `nocover` is used, also when we define `pragma: no cover` (as explained above) in .coverage.py we have to agree on 1 stmt. so, for majority is uses `no cover`

Reply:
Okay, makes sense.